station-renderer-arsenic/src/main/java/net/modificationstation/stationapi/impl/client/arsenic/renderer/aocalc/LightingCalculatorImpl.java

```diff
@@ -97,6 +104,10 @@ public void calculateForQuad(BakedQuad q) {
                 z + Float.intBitsToFloat(q.getVertexData()[26]),
                 q.hasShade()
         );
+        
+        for (byte i = 0; i < light.length; i++) {
+            light[i] = MathHelper.lerp(emission, light[i], 1F);
```

Comment:
Wouldn't `Floats.constrainToRange(light[i], emission, 1)` make more sense here? Or do I not understand the algorithm.

Reply:
The idea is to lerp between normal light (without emission) and with full brightness, that's why I used lerp here

Reply:
It is just a bit more than using minimum value in some range, it actually changes shading of face from normal shading up to lit shading depending on light intensity